hack/lib/test.sh

```diff
@@ -332,26 +336,26 @@ kube::test::version::object_to_file() {
   name=$1
   flags=${2:-""}
   file=$3
-  kubectl version ${flags} | grep "${name} Version:" | sed -e s/"${name} Version: version.Info{"/'/' -e s/'}'/'/' -e s/', '/','/g -e s/':'/'=/g' -e s/'"'/""/g | tr , '\n' > "${file}"
+  kubectl version "${flags}" | grep "${name} Version:" | sed -e s/"${name} Version: version.Info{"/'/' -e s/'}'/'/' -e s/', '/','/g -e s/':'/'=/g' -e s/'"'/""/g | tr , '\n' > "${file}"
```

Comment:
I believe that "${flags}" needs to allow for expansion here. 
```sh
flags="-o yaml --client=true"
kubectl version "${flags}" ## error: --output must be 'yaml' or 'json'
kubectl version ${flags} ## this is OK!
```

Reply:
Fixed.